### DISCUSSION TITLE
问好

### DISCUSSION THREAD

**Discussion by nobodxbodon** [abersheeran/mingshe#8] [General]
之前在 v2 的回复不知是否看到：
木兰语言实现是从代码生成 python AST，也支持三元运算符。重现项目 [开源在此](https://gitee.com/MulanRevive/mulan-rework)

**Comment by abersheeran** [comment]
破案了，我就说 v2ex 上一共七个回复，为什么我只看到了六条，原来你的回复被站长屏蔽了……

**Reply by nobodxbodon** [reply]
> 破案了，我就说 v2ex 上一共七个回复，为什么我只看到了六条，原来你的回复被站长屏蔽了……

原来如此。。难怪跟帖很久没响应了。话说头一次知道issue可以transfer到讨论区，看到个 “closed #7” 的email通知还以为issue被删了。。

**Reply by abersheeran** [reply]
我之前看到有七条回复，但点进去只有六条，我还赶紧去确认我是不是真的没有屏蔽人。😂

**Reply by nobodxbodon** [reply]
嗯，刚试了不登录 [看帖](https://v2ex.com/t/799393#reply7) 的话同样看不到。

**Reply by abersheeran** [reply]
好家伙。第一次遇到这种情况。
